src/kt_variant.h

```diff
@@ -236,12 +236,12 @@ namespace ktvariant {
         //TODO: Manage 32 bits systems.
         des->increment_position(encode_uint64(reinterpret_cast<uintptr_t>(ptr), des->get_cursor()));
 
-        String class_name {ptr->get_class()};
+        StringName class_name{ptr->get_class()};
```

Comment:
@piiertho Is this still needed?

Reply:
Yep, it is an optimization. We were creating the `StringName` from the `String` twice.